src/iiif/ingress_zip_consumer.py

```diff
@@ -27,32 +27,38 @@ class ZipConsumer(BaseConsumer):
     set_consume_started_at = True
 
     def consume_raw_data(self, raw_data):
-        record = json.loads(raw_data)
+        log.info("Started consume_raw_data")
```

Comment:
Misschien ook handig om een info bericht te sturen aan het einde als het gelukt is?

Reply:
Hij eindigd sowieso niet. Het is meer dat ik denk dat hij uberhaupt nooit begint. Ik wil deze sowieso later weer weghalen.